kubetest/gke.go

```diff
@@ -54,6 +54,7 @@ var (
 	gkeCustomSubnet                = flag.String("gke-custom-subnet", "", "(gke only) if specified, we create a custom subnet with the specified options and use it for the gke cluster. The format should be '<subnet-name> --region=<subnet-gcp-region> --range=<subnet-cidr> <any other optional params>'.")
 	gkeReleaseChannel              = flag.String("gke-release-channel", "", "(gke only) if specified, bring up GKE clusters from that release channel.")
 	gkeSingleZoneNodeInstanceGroup = flag.Bool("gke-single-zone-node-instance-group", true, "(gke only) Add instance groups from a single zone to the NODE_INSTANCE_GROUP env variable.")
+	gkeNodePorts                   = flag.String("gke-node-ports", "", "(gke only) List of ports for which custom firewall rule should be created allowing master to connect to nodes. The format should be 'protocol[:port[-port]],[...]' as in gcloud compute firewall-rules create --allow.")
```

Comment:
Correct me if I'm wrong, but this is not fully true. In fact it will create a firewall rule that will open these ports on all nodes to all incoming traffic, not only from masters, right? Please fix

Reply:
Right, comment is wrong.
For the rule that allows only master, we would need separate way of passing master CIDR block, so that gets complicated.
I would go with current approach (same as with Prometheus), fixing the comment. WDYT?